lightning/src/util/config.rs

```diff
@@ -893,3 +893,67 @@ impl Readable for UserConfig {
 		})
 	}
 }
+
+/// Config structure for overriding incoming channel configuration parameters.
+#[derive(Default)]
+pub struct InboundChannelConfigOverrides {
+	/// Overrides the percentage of the channel value we will cap the total value of outstanding inbound HTLCs to. See
+	/// [`ChannelHandshakeConfig::max_inbound_htlc_value_in_flight_percent_of_channel`].
+	pub max_inbound_htlc_value_in_flight_percent_of_channel: Option<u8>,
```

Comment:
Worth adding a builder?

Reply:
Leaving it out then. With the extra level, the number of init fields still remains the same, so I'd say also now no new need for a builder.